documentation/projects/proposals/trust_and_safety/detecting_sensitive_textual_content/20230506-implementation_plan_frontend_blurring_sensitive.md

```diff
@@ -0,0 +1,264 @@
+# 2023-05-06 Implementation Plan: Fetching, blurring sensitive results
+
+**Author**: @dhruvkb
+
+<!-- See the implementation plan guide for more information: https://github.com/WordPress/openverse/tree/19791f51c063d0979112f4b9f4eeace04c8cf5ff/docs/projects#implementation-plans-status-in-rfc -->
+<!-- This template is exhaustive and may include sections which aren't relevant to your project. Feel free to remove any sections which would not be useful to have. -->
+
+## Reviewers
+
+<!-- Choose two people at your discretion who make sense to review this based on their existing expertise. Check in to make sure folks aren't currently reviewing more than one other proposal or RFC. -->
+
+- [ ] @obulat for frontend experience
+- [ ] @zackkrida
+
+## Project links
+
+<!-- Enumerate any references to other documents/pages, including milestones and other plans -->
+
+- [Project Thread](https://github.com/WordPress/openverse/issues/377)
+- [Project Proposal](https://docs.openverse.org/projects/proposals/trust_and_safety/detecting_sensitive_textual_content/20230309-project_proposal_detecting_sensitive_textual_content.html)
+
+## Scope
+
+<!-- An overview of the implementation plan, if necessary. Save any specific steps for the section(s) below. -->
+
+This plan describes the approach, particularly pertaining to the frontend, for
+blurring sensitive content.
+
+The content safety project has an
+[API side](/projects/proposals/trust_and_safety/detecting_sensitive_textual_content/20230309-project_proposal_detecting_sensitive_textual_content.md#api),
+extensively documented in the project proposal but out of the scope for this IP.
+For now, we document what the frontend needs and assume that these needs will be
+met by the API.
```

Comment:
It would be nice to link to the API implementation plan for the project: https://docs.openverse.org/projects/proposals/trust_and_safety/detecting_sensitive_textual_content/20230330-implementation_plan_filtering_and_designating_results_with_sensitive_textual_content.html

I'll cross-check to make sure everything the frontend expects in this IP is covered. If not, we'll need to add any of the dangling items to this IP to be added to the API as well and make a note that the requirement was missed during the initial planning that split the project into API and frontend IPs.

Reply:
Linked to API IP in 15919db.